pkg/js/helpers.js

```diff
@@ -156,6 +170,18 @@ function ALIAS(name, target) {
     }
 }
 
+// CAA(name,tag,value, recordModifiers...)
+function CAA(name, tag, value){
+    checkArgs([_.isString, _.isString, _.isString], arguments, "CAA expects (name, tag, value)")
```

Comment:
The flag is implemented as a record modifier (eg. `CAA_CRITICAL`) since it is optional. Should I add it to the function definition instead?